apps/s_time.c

```diff
@@ -76,6 +76,10 @@ const OPTIONS s_time_options[] = {
 #ifndef OPENSSL_NO_SSL3
     {"ssl3", OPT_SSL3, '-', "Just use SSLv3"},
 #endif
+    {"tls1", OPT_TLS1, '-', "Just use TLSv1.0"},
```

Comment:
Just like ssl3 there are OPENSSL_NO_ variants for TLS1 etc..
See apps/ciphers.c for example.